src/main/resources/META-INF/resources/primefaces/core/core.csp.js

```diff
@@ -63,7 +69,29 @@ if (!PrimeFaces.csp) {
                 };
 
                 $(element).off(jqEvent).on(jqEvent, jsWrapper);
+
+                //TODO: Depending on some (debug-)switch?
```

Comment:
Yes add `if (PrimeFaces.isDevelopmentProjectStage()) {` so this only runs when JSF stage is development.

Reply:
Are we sure integration-tests based on PrimeFaces Selenium are only executed in JSF development-stage?